flowetl/dags/etl.py

```diff
@@ -9,90 +9,24 @@
 import logging
 import os
 
-from airflow import DAG
-from airflow.models import DagRun, TaskInstance
-from airflow.operators.python_operator import BranchPythonOperator, PythonOperator
+from airflow import DAG  # pylint: disable=unused-import
```

Comment:
I assume you're purposely keeping the import, but what for out of interest?

Reply:
Airflow doesn't even look in the file for a DAG if it's not imported...

Reply:
Oh gosh... 🤕 

Can we please add a comment to this effect then, otherwise I can totally see someone (probably me...) accidentally cleaning this up in the future and breaking everything as a consequence.

Reply:
Already have!